js/home.js

```diff
@@ -9,204 +9,247 @@ weekInfoContainer.className = "container";
-      cityName = weatherInfo.name;
-      fetch(
-        `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=minutely,hourly,alerts&appid=${apiKey}`
-      )
-        .then((info) => info.json())
-        .then((db) => {
-          currentDayData = db.current;
-          dailyData = db.daily.slice(1, 7);
-          renderCurrentDayData(currentDayData, countryName, cityName);
-          renderDailyData(dailyData);
-        });
-    })
-    // If the search term is INVALID, a popup will show to ask the user to re-write a VALID input
-    .catch((err) => placeNotFound(searchTerm));
+    let lat,
+        lon,
+        countryName,
+        cityName,
+        currentDayData,
+        dailyData,
+        apiKey = "d0139168498c4f66d3fb31b4d374f145";
+    fetch(
+            `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&appid=${apiKey}`
+        )
+        .then((res) => res.json())
+        .then((weatherInfo) => {
+            lat = weatherInfo.coord.lat;
+            lon = weatherInfo.coord.lon;
+            countryName = weatherInfo.sys.country;
+            cityName = weatherInfo.name;
+            fetch(
+                    `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=minutely,hourly,alerts&appid=${apiKey}`
+                )
+                .then((info) => info.json())
+                .then((db) => {
+                    console.log(db)
+                        ////////////////////////////
+                    let currente = db.current.weather[0].main //give the weather condition 
+                        //images show According to the weather 
+                    let images = {
+                            Rain: ["../assets/images/rainy/1.jpg", "../assets/images/rainy/2.jpg", "../assets/images/rainy/3.jpg", "../assets/images/rainy/4.jpg"],
+                            Clear: "../assets/images/clear/1.jpg",
+                            Clouds: ["../assets/images/cloudy/1.jpg", "../assets/images/cloudy/2.jpg", "../assets/images/cloudy/3.jpg"],
+                            Sunny: ["../assets/images/sunny/1.jpg", "../assets/images/sunny/2.jpg", "../assets/images/sunny/3.jpg", "../assets/images/sunny/4.jpg"],
+                            Snow: ["../assets/images/snowy/1.jpg", "../assets/images/snowy/2.jpg", "../assets/images/snowy/3.jpg", "../assets/images/snowy/4.jpg"],
+                            Windy: ["../assets/images/windy/1.jpg", "../assets/images/windy/2.jpg", "../assets/images/windy/3.jpg", "../assets/images/windy/4.jpg"]
+                        }
+                        //Compare the weather with the key of object
+                    let describ = "";
+                    switch (true) {
+                        case currente == "Clouds":
+                            describ = images.Clouds
+                            break;
+                        case currente == "Rain":
+                            describ = images.Rain
```

Comment:
use [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to format your code so the page will look neat